src/OrbitGl/TrackManagerTest.cpp

```diff
@@ -16,27 +16,6 @@ using orbit_client_protos::TimerInfo;
 
 namespace orbit_gl {
 
-class UnitTestTrack : public Track {
```

Comment:
@reichlfl I thing you add this UnitTestTrack that it's not being used I suppose. Do you have any complain with this?

Reply:
Ah, right, that may no longer be needed. I thought I'd need it for the tests but ended up using the regular tracks, so removing this is fine.